apps/screener-proxy/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "screener-proxy",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "build": "tsc",
+    "start": "node dist/index",
+    "dev": "nodemon index.ts"
+  },
+  "dependencies": {
+    "@octokit/app": "10.1.0",
+    "body-parser": "^1.19.0",
+    "express": "4.17.1"
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.11",
+    "@types/node": "^14.14.27",
+    "dotenv": "^8.2.0",
+    "ngrok": "^3.4.0",
+    "nodemon": "^2.0.7",
+    "prettier": "^2.2.1",
+    "typescript": "^4.1.5"
```

Comment:
Please use the same versions of prettier and typescript that are used elsewhere in the repo. Introducing different versions creates problems. (And actually in this case I don't think you need the prettier dep, that was probably only for when it was a separate repo.)